src/runtime/cmd/kata-runtime/kata-check_amd64_test.go

```diff
@@ -322,6 +325,30 @@ func TestCheckHostIsVMContainerCapable(t *testing.T) {
 	err = hostIsVMContainerCapable(details)
 	assert.Nil(err)
 
+	// Remove required kernel modules and add them to denylist
+	denylistFile, err := os.Create(denylistModuleConf)
+	assert.Nil(err)
+	succeedToRemoveOneModule := false
+	for mod := range archRequiredKernelModules {
+		cmd := exec.Command(modProbeCmd, "-r", mod)
+		if output, err := cmd.CombinedOutput(); err == nil {
+			succeedToRemoveOneModule = true
+		} else {
+			kataLog.WithField("output", string(output)).Warn("failed to remove module")
+		}
+		// Write the following into the denylist file
+		// blacklist <mod>
+		// install <mod> /bin/false
+		_, err = denylistFile.WriteString(fmt.Sprintf("blacklist %s\ninstall %s /bin/false\n", mod, mod))
```

Comment:
And here as well. :-)

Reply:
This can't be modified to `denylist` since it is a invalid syntax for libkmod.

```bash
$ cat /etc/modprobe.d/denylist-kata-kernel-modules.conf
denylist vhost-net
$ sudo modprobe vhost-net
libkmod: ERROR ../libkmod/libkmod-config.c:712 kmod_config_parse: /etc/modprobe.d/denylist-kata-kernel-modules.conf line 1: ignoring bad line starting with 'denylist'
```